cmd/kubelet/app/server.go

```diff
@@ -155,11 +156,20 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 		Run: func(cmd *cobra.Command, args []string) {
 			// initial flag parse, since we disable cobra's flag parsing
 			if err := cleanFlagSet.Parse(args); err != nil {
-				klog.ErrorS(err, "Failed to parse kubelet flag")
+				fmt.Fprintf(cmd.OutOrStderr(), "Failed to parse kubelet flag err: %v", err)
```

Comment:
I would expect the initLogs call in main() to make klog reasonable to rely on here... needing to use stderr/stdout until config is parsed is difficult to reason about

what is the current behavior calling klog.ErrorS here?

Reply:
It will use default klog configuration so write to stderr in klog format.

Reply:
using `fmt.Fprintf(cmd.OutOrStderr()` doesn't seem meaningfully different, and is harder to reason about

Reply:
If parse flagset failed, but klog format set json succed, this will result in: this line is text format, but other lines is json format.

Reply:
if parsing the flagset fails, the command won't start successfully anyway, right?